tests/config/opentelemetry-operator/crd/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+- bases/opentelemetry.io_opentelemetrycollectors.yaml
```

Comment:
I need this to install the CDR for testEnv, otherwise I'll get the error: " no matches for kind "OpenTelemetryCollector" in version "opentelemetry.io/v1alpha1"